renovate.json

```diff
@@ -0,0 +1,71 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": [
+		"config:base"
+	],
+	"timezone": "Europe/Berlin",
+	"schedule": [
+		"every weekend"
+	],
+	"reviewers": ["team:office"],
+	"labels": ["dependencies"],
+	"rangeStrategy": "bump",
+	"rebaseWhen": "conflicted",
+	"ignoreUnstable": false,
+	"baseBranches": [
+		"main",
+		"stable27",
+		"stable26",
+		"stable25"
+	],
+	"enabledManagers": [
+		"npm"
+	],
+	"ignoreDeps": [
+		"node",
+		"npm"
+	],
+	"packageRules": [
+		{
+			"groupName": "cypress",
+			"matchPackageNames": [
+				"cypress",
+				"@nextcloud/cypress",
+				"@cypress/"
+			]
+		},
+		{
+			"groupName": "tiptap",
+			"matchPackagePrefixes": [
+				"@tiptap/",
+				"prosemirror-"
+			]
+		},
+		{
+			"matchUpdateTypes": ["major"],
+			"matchBaseBranches": ["stable27", "stable26", "stable25"],
```

Comment:
Remove those and other stable* references as long as we don't actively maintain those stable branches?